### DISCUSSION TITLE
SD 2.0 scripts can train SD 1.5 models?

### DISCUSSION THREAD

**Discussion by elismasilva** [bghira/SimpleTuner#347] [General]
Can this script train sd 1.5 models?

**Comment by bghira** [comment]
most likely. i don't think there's any hardcoded shapes. but i have zero interest in SD 1.x and never tested it.

**Reply by elismasilva** [reply]
well i will test it, if it not work then i have a script of huggingface working for sd 1.5 dreambooth and deepspeed, may i can try adapt this with your script and add new support for sd 1.5 dreambooth too.

**Reply by bghira** [reply]
i'm not terribly interested in fixing whatever issues are needed for SD 1.5 models to be trained. i was never a fan of SD 1.5 or its finetunes, and it requires a lot of post work to make it look good.

in fact, at some point, i will likely stop training stable diffusion models at all, and that portion of the codebase will be put on life support.

with the arrival of pixart sigma, it's looking like that's to happen sooner than later
